client/src/components/Button.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from "react";
+import { Button as BootstrapButton } from "react-bootstrap";
+
+export const Button = ({text}) => {
+  const [hover, setHover] = useState(false);
+
+  const toggleHover = () => {
+    setHover(!hover);
```

Comment:
`setHover(!hover)` can more specifically access [previous state](https://reactjs.org/docs/hooks-reference.html#functional-updates) here to update like this: `setHover(previousHoverValue => !previousHoverValue)`